cloudlink-combined-storage/README.md

```diff
@@ -0,0 +1,79 @@
+# CloudLink-Combined-Storage
+
+A JavaFX application that uses Gluon Mobile Glisten (a mobile application framework with mobile-friendly UI controls) and 
+Gluon CloudLink to manage storage of notes both locally in the device and remotely in the cloud, with a combined strategy.
+
+## Prerequisites
+
+You need a valid subscription to Gluon CloudLink. You can get it [here](http://gluonhq.com/products/cloudlink/buy/), and 
```

Comment:
This section needs an update. CloudLink projects are now managed via gluon.io instead of the Dashboard app.